include/BandLimitedWave.h

```diff
@@ -94,79 +89,72 @@ class LMMS_EXPORT BandLimitedWave
 		NumBLWaveforms
 	};
 
-	BandLimitedWave() {};
-	virtual ~BandLimitedWave() {};
+	BandLimitedWave(){};
+	virtual ~BandLimitedWave(){};
 
 	/*! \brief This method converts frequency to wavelength. The oscillate function takes wavelength as argument so
 	 * use this to convert your note frequency to wavelength before using it.
 	 */
-	static inline float freqToLen( float f )
-	{
-		return freqToLen( f, Engine::audioEngine()->processingSampleRate() );
-	}
+	static inline float freqToLen(float f) { return freqToLen(f, Engine::audioEngine()->processingSampleRate()); }
 
 	/*! \brief This method converts frequency to wavelength, but you can use any custom sample rate with it.
 	 */
-	static inline float freqToLen( float f, sample_rate_t sr )
-	{
-		return static_cast<float>( sr ) / f;
-	}
+	static inline float freqToLen(float f, sample_rate_t sr) { return static_cast<float>(sr) / f; }
 
 	/*! \brief This method converts phase delta to wavelength. It assumes a phase scale of 0 to 1. */
-	static inline float pdToLen( float pd )
-	{
-		return 1.0f / pd;
-	}
+	static inline float pdToLen(float pd) { return 1.0f / pd; }
 
 	/*! \brief This method provides interpolated samples of bandlimited waveforms.
 	 *  \param _ph The phase of the sample.
-	 *  \param _wavelen The wavelength (length of one cycle, ie. the inverse of frequency) of the wanted oscillation, measured in sample frames
-	 *  \param _wave The wanted waveform. Options currently are saw, triangle, square and moog saw.
+	 *  \param _wavelen The wavelength (length of one cycle, ie. the inverse of frequency) of the wanted oscillation,
+	 * measured in sample frames \param _wave The wanted waveform. Options currently are saw, triangle, square and moog
```

Comment:
:vomiting_face:

Seeing this, I'm more and more against clang-format...

Reply:
Can be fixed by `CommentPragmas:  '^\\.+'`, but I need to revert all the code files again because now, the code is already broken.